### PR TITLE
[Catalog] image size and pattern alignment Fixed

### PR DESCRIPTION
**Pull request by MrBehl06**
Screenshot 2023-01-12 at 12 55 24

This PR fixes #
#951 

1) The Image size issue fixed
2) Box Modal Size and modal-pattern alignment fixed 

Extra Following Changes
1) Commented out the space cutlets
2) Commented out the repeated classes

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: _sass/catalog.scss [hunk 1]
```diff
@@ -455,12 +455,11 @@ input[type="checkbox"] {
   }
 }
 .modal-pattern-name {
-  color: #333;
-  text-align: center;
+color: #333;
+  margin-left: 40px;
```
Comment (GaganpreetKaurKalsi) [inline _sass/catalog.scss:459]: - add indentation to color like before
- remove margin-left as it is affecting the title positioning 
image

File: _sass/catalog.scss [hunk 2]
```diff
@@ -501,20 +500,19 @@ input[type="checkbox"] {
...
-  bottom: 0;
-  font-size: 8px;
-  right: 0;
-  padding-top: 10px;
-  margin-right: 14px;
+	bottom: 0;
+	font-size: 8px;
+	right: 0;
+	margin: 0px 62px 4px 0px;
+
```
Comment (GaganpreetKaurKalsi) [inline _sass/catalog.scss:515]: fix indentations

**Review by GaganpreetKaurKalsi — CHANGES_REQUESTED** [review]

File: _sass/catalog.scss (hunk 2 above)
Comment (MrBehl06) [reply _sass/catalog.scss:515]: Okey

**Comment by MrBehl06** [comment]
@GaganpreetKaurKalsi @MrBehl06 Please review the changes. And if there is any issue do let me know

**Review by GaganpreetKaurKalsi — APPROVED** [review]
Looks good 👍 🚀 Thanks @MrBehl06

**Review by leecalcote — APPROVED** [review]
Way to drive this home, @MrBehl06 and @GaganpreetKaurKalsi 👏

**Pull request merged**